lib/_parse-gerber.js

```diff
@@ -96,6 +96,8 @@ var parseToolDef = function(parser, block) {
 var parseMacroDef = function(parser, block) {
   var macroMatch = block.match(reMACRO)
   var name = macroMatch[1]
+  if( name.match(/\-/) )
```

Comment:
This if block does not pass style checking. Please run `$ npm run lint` before pushing